stdlib_tests/test_factorial.py

```diff
@@ -0,0 +1 @@
+print(factorial(3))
```

Comment:
Some TranspileTestCases would be nice in order to test different types of input to the factorial function. This would ensure that the function behaves in the same way in a native Python environment as in Java for different inputs. 

See for example _voc/tests/datatypes/test_bool.py_ for inspiration.

Reply:
Thanks! Gonna look into it